src/arrays.js

```diff
@@ -1,77 +1,94 @@
 const getNthElement = (index, array) => {
-  // your code here
+    if (index < array.length) {
+        return array[index]
+    } else {
+        return array[index - array.length]
+    }
+    // your code here
 };
 
 const arrayToCSVString = (array) => {
-  // your code here
+    return array.toString(array)
+        // your code here
 };
 
 const csvStringToArray = (string) => {
-  // your code here
+    return string.split(',') // your code here
 };
 
 const addToArray = (element, array) => {
-  // your code here
+    array.push(element) // your code here
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+    return array.concat(element) // your code here
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+    array.splice(index, 1)
 };
 
 const numbersToStrings = (numbers) => {
-  // your code here
+    return numbers.toString(numbers).split(',')
+        // your code here
 };
 
 const uppercaseWordsInArray = (strings) => {
-  // your code here
+    return strings.map(a => a.toUpperCase())
 };
 
 const reverseWordsInArray = (strings) => {
-  // your code here
+    return strings.map(a => a.split("").reverse().join(""))
 };
 
 const onlyEven = (numbers) => {
-  // your code here
+    return numbers.filter(num => num = 0 || num % 2 === 0)
```

Comment:
The first part of your expression: `num = 0` is assignment, not equality (one `=` denotes assignment, `===` denotes strict equality).